policy/release/lib/attestations_test.rego

```diff
@@ -49,3 +56,65 @@ test_results_from_tests {
 	expected := {"result": "SUCCESS", "foo": "bar", "__task_name": "mytask"}
 	assert_equal([expected], results_from_tests) with input.attestations as [att_mock_helper({"result": "SUCCESS", "foo": "bar"}, "mytask")]
 }
+
+test_task_in_pipelinerun {
+	task_name := "my-task"
+	d := att_mock_task_helper({"name": task_name})
+
+	task_in_pipelinerun(task_name) with input.attestations as d
```

Comment:
Should we check that the `task_in_pipelinerun` returns the task that we expect it to return?

Something along the lines of:

```suggestion
	assert_equal({"name": task_name}, task_in_pipelinerun(task_name)) with input.attestations as d
```